python/styleguide.md

```diff
@@ -0,0 +1,3 @@
+# Python Code Style Guide
+
+Hello, World!
```

Comment:
I disagree. Hello World should not be the only thing I'm allowed to put into my code.

:)